src/node/mini_miner.cpp

```diff
@@ -71,8 +71,11 @@ MiniMiner::MiniMiner(const CTxMemPool& mempool, const std::vector<COutPoint>& ou
 
     // Add every entry to m_entries_by_txid and m_entries, except the ones that will be replaced.
```

Comment:
Changes in this commit is identical to the one in https://github.com/bitcoin/bitcoin/pull/28762/commits/d525998b07be066c97a36536b58f0b98d816be39 , but here the `txmempool.h` dependency from `mini_miner.h`, and the imports are not updated?